apps/api/src/mail/services/interface.service.ts

```diff
@@ -10,16 +10,17 @@ export interface IMailService {
     workspace: string,
     actionUrl: string,
     invitedBy: string,
-    role: WorkspaceRole
+    role: WorkspaceRole,
+    forRegisteredUser: boolean
```

Comment:
Change the name of the function to something more meaningful since we are now adding this flag!